### PR TITLE
Update to Oculus SDK 0.7.0.0-beta

### PR DESCRIPTION
**Pull request by Squareys**
Hello everybody!

This is a work-in-progress pullrequest for the update of the former `LibOvrIntegration` (now `OvrIntegration`), mentioned in #8 .

TODOs left:
- [x] Rename LibOvrIntegration to OvrIntegration
- [x] Adapt to Oculus SDK API
- [x] Adapt Example to API changes
- [x] Support DebugHud
- [x] Add Debug Hud to example

Greetings, Squareys

### CODE REVIEW

File: src/Magnum/OvrIntegration/visibility.h [hunk 1]
```diff
@@ -28,10 +28,10 @@
 
 #include <Corrade/Utility/VisibilityMacros.h>
 
-#ifdef MagnumLibOvrIntegration_EXPORTS
-    #define MAGNUM_LIBOVRINTEGRATION_EXPORT CORRADE_VISIBILITY_EXPORT
+#ifdef MagnumOvrIntegration_EXPORTS
+    #define MAGNUM_OvrIntegration_EXPORT CORRADE_VISIBILITY_EXPORT
```
Comment (mosra) [inline src/Magnum/OvrIntegration/visibility.h]: Let's keep it uppercase here (`MAGNUM_OVRINTEGRATION_EXPORT`).

Comment (Squareys) [inline src/Magnum/OvrIntegration/visibility.h]: Thanks for catching that, that was Qt Creators Find/Replace.

**Comment by Squareys** [comment]
This is very close to done. I will go through the code for some cleanup tomorrow.

**Comment by Squareys** [comment]
This and https://github.com/mosra/magnum-examples/pull/15 are both ready for merge from my side.

File: src/Magnum/OvrIntegration/Test/ConversionTest.cpp [hunk 2]
```diff
@@ -30,9 +30,9 @@
 #include "Magnum/Math/Matrix3.h"
 #include "Magnum/Math/DualQuaternion.h"
 
-#include "Magnum/LibOVRIntegration/Conversion.h"
+#include "Magnum/OVRIntegration/Conversion.h"
```
Comment (mosra) [inline src/Magnum/OvrIntegration/Test/ConversionTest.cpp]: `Ovr` (so it works on case sensitive filesystems)

Comment (Squareys) [inline src/Magnum/OvrIntegration/Test/ConversionTest.cpp]: Uh, I don't know why that was ever `LibOVRIntegration` in the first place.

**Comment by Squareys** [comment]
I will be able to look after the issues you pointed out this evening.

**Comment by Squareys** [comment]
@mosra Alright, those two issues are done.

**Comment by Squareys** [comment]
I need to fix the doc, give me another half an hour, then I should be *really* done.

**Comment by Squareys** [comment]
Okay, ready :)

**Comment by Squareys** [comment]
@mosra If there are any other issues, just let me know :)

**Comment by mosra** [comment]
Merged except for the NoInit commit (Squareys/magnum-integration@a27713ebb52220c322eaf9f89ccc351ec9d16edc), will merge that one too when the crash mentioned above is resolved.

**Comment by mosra** [comment]
@Squareys: latest Corrade (mosra/corrade@8d1b47e818b924042dd516700d462f6d9b64f845) has custom deleters for Array, which should fix the NoInit issue. Can you please verify if the crash is gone now? You may need to rebuild Magnum too.

Thanks a lot.

**Comment by Squareys** [comment]
@mosra The crash is gone! Thank you, too!

**Comment by mosra** [comment]
Awesome, closing as merged.

**Pull request closed without merging**